docs/client_api/filtering/Conditions/Gte.md

```diff
@@ -0,0 +1,16 @@
+::: valor.schemas.Gte
```

Comment:
could all of the "Conditions" exist on one page, and all of the "Logic" on another, rather than having a separate page for each individual class? in this setup, it seems like it would take a lot of clicking between pages for a user to figure out what to use. some of the docs for these classes (like most of the "Conditions") are also very barebones, so it seems odd to break them out separately

Reply:
![image](https://github.com/user-attachments/assets/d872a1d6-d324-4cc4-b908-721978d14938)

I think this leads to a confusing docs layout. Do you have any suggestions on how to improve this?

Reply:
I don't think that format is visually confusing, but it's weird that you're mixing operators (Gt, Gte, etc.) with other stuff (Condition, Function) under a header called "Operators". maybe move these latter two docstrings into Filter or another category?